weblogic-azure-aks/src/main/resources/marketing-artifacts/README.md

```diff
@@ -0,0 +1,63 @@
+<p>There are a number of offers that support different common use cases such as a single working instance with Admin Server enabled as well as highly available clusters.</p>
+<p>The offers are linked in the <i>Learn more</i> section at the bottom of this page. You should feel free to give them a try, the offers are available free of charge.</p>
+<p><i>These offers are Bring-Your-Own-License</i>. They assume you have already procured the appropriate licenses with Oracle and are properly licensed to run offers in Microsoft Azure.</p>
+<p>This offer supports a range of operating system, Java and WebLogic versions such as WLS 14 and JDK 11 on Oracle Linux 7.6 through base images. These base images are also available on Azure on their own. The base images are suitable for customers that require very highly customized Azure deployments. The current set of base images are available <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps?search=oracle%20weblogic%20base%20image&amp;page=1">in the Azure marketplace</a>.<br></p>
+<h6>WLS Single Node</h6>
+<p>This offer provisions a single virtual machine and installs WLS on it. It does not create a domain or start the Administration Server. This is useful for scenarios with highly customized domain configuration.</p>
+<h6>WLS with Admin Server</h6>
+<p>This offer provisions a single virtual machine and installs WLS on it. It creates a domain and starts up the Administration Server, which allows you to manage the domain.</p>
+<h6>WLS Cluster</h6>
+<p>This offer creates a highly available cluster of WLS virtual machines. The Administration Server and all managed servers are started by default, which allow you to manage the domain.</p>
+<h6>WLS Dynamic Cluster</h6>
+<p>This offer creates a highly available and scalable dynamic cluster of WLS virtual machines. The Administration Server and all managed servers are started by default, which allow you to manage the domain.</p>
+
+<h2>Search keywords</h2>
+
+Java
+Java EE
+WebLogic
+
+<h2>Privacy policy link</h2>
+
+https://www.oracle.com/legal/privacy/privacy-policy.html
+
+<h2>Useful links</h2>
+
+[Overview](https://www.oracle.com/middleware/weblogic/)
+
+[Oracle WebLogic Server Single Node](https://portal.azure.com/#create/oracle.20191001-arm-oraclelinux-wls20191001-arm-oraclelinux-wls)
+
+[Oracle WebLogic Server With Admin Server](https://portal.azure.com/#create/oracle.20191009-arm-oraclelinux-wls-admin20191009-arm-oraclelinux-wls-admin)
+
+[Oracle WebLogic Server Cluster](https://portal.azure.com/#create/oracle.20191007-arm-oraclelinux-wls-cluster20191007-arm-oraclelinux-wls-cluster)
+
+[Oracle WebLogic Server Dynamic Cluster](https://portal.azure.com/#create/oracle.20191021-arm-oraclelinux-wls-dynamic-cluster20191021-arm-oraclelinux-wls-dynamic-cluster)
+
+[Fusion Middleware Documentation](https://docs.oracle.com/en/middleware/fusion-middleware/index.html)
+
+[Learn More](https://www.oracle.com/middleware/technologies/weblogic.html)
```

Comment:
You don't need to keep this link here because I reproduced it earlier in the doc, but you can keep it here, too, if you want.